control/strategy/pool.go

```diff
@@ -297,6 +298,20 @@ func (p *pool) Kill(id uint32, reason string) {
 	}
 }
 
+// Kill all instances of a plugin
+func (p *pool) KillAll(reason string) {
+	for _, rp := range p.plugins {
+		rp.Stop(reason)
```

Comment:
Looking at it, would it make sense to do this for the loop instead to avoid duplication?

```go
for id := range p.plugins {
    p.SelectAndKill(rp, reason)
}
```